addons/event/static/src/client_action/event_barcode.scss

```diff
@@ -24,6 +24,34 @@
         max-width: 200px;
         max-height: 100px;
     }
+    .o_barcode_mobile_container {
+        transform: scale(2.5);
```

Comment:
Also note that the margins below will also be scaled up. Since we also add a height on the image below, should we not directly set heights and margins instead of using transform? Maybe I am missing something though

Reply:
I was just randomly trying values :sweat_smile: and doing `transfrom: scale(2.5);` perfectly matched.
But now I did it with just height and width.